exonum/src/runtime/rust/mod.rs

```diff
@@ -12,7 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Built-in Rust runtime module.
+//! The basic runtime with the native Rust services.
+//!
+//! This runtime is usually presents in every blockchain instance and can only process a static set
+//! of available Rust services. To change this set you have to recompile the node binary.
+//! During this procedure, be very careful and avoid situations when some of deployed services do
```

Comment:
I'd rephrase like "Beware of the removing the artifacts from the Rust runtime, since attempt to remove an artifact with already running instances can cause blockchain to break. The only safe change that can be performed here is adding new artifacts."